internal-packages/react-graphql-test-app/src/main.tsx

```diff
@@ -20,6 +21,7 @@ interface Entity {
 export const client = createClient({
   url: 'http://localhost:4000/graphql',
   id: (v: Entity) => `${v.__typename}:${v.id}`,
+  fetch: buildFetch([]),
```

Comment:
I think that this use could use the default value instead right? I assume if you don't provide a `fetch` we'll use the native / `globalThis.fetch` right?